src/main/java/edu/ie3/datamodel/io/connectors/CsvFileConnector.java

```diff
@@ -243,6 +249,13 @@ public BufferedReader initReader(String filePath) throws FileNotFoundException {
       IndividualTimeSeriesMetaInformation individualMetaInformation =
           (IndividualTimeSeriesMetaInformation) metaInformation;
 
+      if (columnSchemes != null
+          && columnSchemes.length > 0
+          && Stream.of(columnSchemes)
+              .noneMatch(scheme -> scheme.equals(individualMetaInformation.getColumnScheme()))) {
+        return Optional.empty();
```

Comment:
You might consider making this a little bit more verbose by adding a logged warning or error?